velox/docs/functions/spark/aggregate.rst

```diff
@@ -39,7 +39,7 @@ General Aggregate Functions
 
 .. spark:function:: bloom_filter_agg(hash) -> varbinary
     
-    A version of ``bloom_filter_agg`` that use 4,194,304 as ``numBits``.
+    A version of ``bloom_filter_agg`` that use config default value 4,194,304 as ``numBits``.
```

Comment:
Does this function use the config value or hard-coded value? Would be nice to clarify.